compression/2.2/modules/ROOT/pages/compression-module-compress-decompress-example.adoc

```diff
@@ -0,0 +1,59 @@
+
+
+= Compress and Decompress Example
+
+The following Compression module examples show how to compress a file using the *Compress* operation and decompress a payload from a remote service using the *Decompress* operation.
+
+== Compress a File
+
+In the following example, you configure the File Connector *Read* operation to read a file. Then, the *Compress* operation compresses the file and saves it in `GZip` format. Subsequently, the File Connector *Write* operation writes the content into the given path on demand.
+
+. In Studio, drag the File Connector *Read* operation to your flow.
+. Set *Path* to `file.txt`.
+. Drag the *Compress* operation to the right of the *Read* operation.
```

Comment:
I saw something in our accessibility guidelines about avoiding directional things like "right and left" 

Is is possible to revise to something like, Drag the *Compress* operation alongside the *Read* operation. 

Or ..into the space next to the *Read* operation

I'm not going to worry too much about this. I think it's just something to consider, I think. Also, apologies if I didn't call this out earlier.